src/styles/globals.css

```diff
@@ -1,188 +1,65 @@
-    --secondary-glow: linear-gradient(
-      to bottom right,
-      rgba(1, 65, 255, 0),
-      rgba(1, 65, 255, 0),
-      rgba(1, 65, 255, 0.3)
-    ); */
-
-    --tile-start-rgb: 2, 13, 46;
-    --tile-end-rgb: 2, 5, 19;
-    --tile-border: conic-gradient(
-      #ffffff80,
-      #ffffff40,
-      #ffffff30,
-      #ffffff20,
-      #ffffff10,
-      #ffffff10,
-      #ffffff80
-    );
-
-    --callout-rgb: 20, 20, 20;
-    --callout-border-rgb: 108, 108, 108;
-    --card-rgb: 100, 100, 100;
-    --card-border-rgb: 200, 200, 200;
-  }
+  --primary: #CF1FB1;
+  --background-dark: #0E001A;
+  --text-dark: #3D4551;
 }
 
 @font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-Bold.otf') format('opentype');
+  font-family: 'Sharp Sans Display';
+  src: url('/fonts/SharpSansDisplay-Bold.otf') format('opentype');
   font-weight: 700;
   font-style: normal;
 }
 
 @font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-BoldItalic.otf') format('opentype');
-  font-weight: 700;
-  font-style: italic;
+  font-family: 'Sharp Sans Display';
+  src: url('/fonts/SharpSansDisplay-Medium.otf') format('opentype');
+  font-weight: 500;
+  font-style: normal;
 }
 
 @font-face {
   font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-Book.otf') format('opentype');
-  font-weight: 400;
+  src: url('/fonts/SharpSans-Bold.otf') format('opentype');
+  font-weight: 700;
   font-style: normal;
 }
 
 @font-face {
   font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-BookItalic.otf') format('opentype');
+  src: url('/fonts/SharpSans-Book.otf') format('opentype');
   font-weight: 400;
-  font-style: italic;
-}
-
-@font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-Extrabold.otf') format('opentype');
-  font-weight: 800;
   font-style: normal;
 }
 
-@font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-ExtraboldItalic.otf') format('opentype');
-  font-weight: 800;
-  font-style: italic;
-}
-
 @font-face {
   font-family: 'Sharp Sans';
   src: url('/fonts/SharpSans-Light.otf') format('opentype');
   font-weight: 300;
   font-style: normal;
 }
 
-@font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-LightItalic.otf') format('opentype');
-  font-weight: 300;
-  font-style: italic;
-}
-
 @font-face {
   font-family: 'Sharp Sans';
   src: url('/fonts/SharpSans-Medium.otf') format('opentype');
   font-weight: 500;
   font-style: normal;
 }
 
-@font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-MediumItalic.otf') format('opentype');
-  font-weight: 500;
-  font-style: italic;
-}
-
-@font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-Semibold.otf') format('opentype');
-  font-weight: 600;
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Sharp Sans';
-  src: url('/fonts/SharpSans-SemiboldItalic.otf') format('opentype');
-  font-weight: 600;
-  font-style: italic;
-}
-
```

Comment:
You've removed these fonts from the global.css but they are still in the repo, adding unnecessary bloat. The only fonts in `public/fonts` should be ones that are actually used.